src/main/java/codeu/controller/ProfilePageServlet.java

```diff
@@ -0,0 +1,40 @@
+package codeu.controller;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+/** Servlet class responsible for a user's profile page. */
+public class ProfilePageServlet extends HttpServlet {
+
+    /**
+     * This function fires when a user navigates to a profile page. It displays a user's About Me section
+     * and depending on if the profile is the current user's, allows editing of the About Me section.
+     */
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+    	String user = (String) request.getSession().getAttribute("user");
+    	String author = request.getRequestURI().substring("/profile/".length());
+    	
```

Comment:
If you manually type in /profile/abc in the browser (assuming abc is a user that isn't registered), does it show a valid profile page too? This might be something for the MVP but I'm thinking you might have to get the UserStore and check if the user is valid first using UserStore's getUser() method

Reply:
Great point. I changed it so that it just redirects to the home page if a user is not found for now. I will try and do a page where it just says that the user is not found for the MVP.